dropwizard-app/pom.xml

```diff
@@ -42,9 +42,30 @@
         </dependency>
 
         <dependency>
+            <groupId>com.smoketurner</groupId>
+            <artifactId>dropwizard-swagger</artifactId>
+            <version>1.0.5-3</version>
```

Comment:
Figure it out.  The com.wordnik stuff was moved do io.swagger and revved the version to match that of swagger-core.

The old dropwizard-swagger dependency brought a lot of transitive dependencies that had the planets align to just work.  Now we need to manage them ;).